tests/test_search/test_elastic.py

```diff
@@ -78,152 +58,149 @@ def wrapped(*args, **kwargs):
     return test_wrapper
 
 
-@requires_search
 class TestUserUpdate(SearchTestCase):
 
     def setUp(self):
         super(TestUserUpdate, self).setUp()
         search.delete_index(elastic_search.INDEX)
         search.create_index(elastic_search.INDEX)
-        self.user = UserFactory(fullname='David Bowie')
+        self.user = factories.UserFactory(fullname='David Bowie')
 
     def test_new_user(self):
         # Verify that user has been added to Elastic Search
         docs = query_user(self.user.fullname)['results']
-        assert_equal(len(docs), 1)
+        assert len(docs) == 1
```

Comment:
I'm afraid this change is not approved. Aside from the stylistic difference, the nose_asserts give more information than the bare asserts unless we are using pytest (which we do not). Please revert these back.